contracts/access/AccessControlDefaultAdminRules.sol

```diff
@@ -0,0 +1,244 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v4.8.0) (access/AccessControlDefaultAdminRules.sol)
+
+pragma solidity ^0.8.0;
+
+import "./AccessControl.sol";
+import "./IAccessControlDefaultAdminRules.sol";
+import "../utils/math/SafeCast.sol";
+import "../interfaces/IERC5313.sol";
+
+/**
+ * @dev Extension of {AccessControl} that allows specifying special rules to manage
+ * the `DEFAULT_ADMIN_ROLE` holder, which is a sensitive role with special permissions
+ * over other roles that may potentially have privileged rights in the system.
+ *
+ * If a specific role doesn't have an `adminRole` assigned, the holder of the
+ * `DEFAULT_ADMIN_ROLE` will have the ability to manage it, as determined by the
+ * function {getRoleAdmin}'s default value (`address(0)`).
+ *
+ * This contract implements the following risk mitigations on top of the {AccessControl} implementation:
+ *
+ * * Only one account holds the `DEFAULT_ADMIN_ROLE` at every time after construction except when it's renounced.
+ * * Enforce a 2-step process to transfer the `DEFAULT_ADMIN_ROLE` to another account.
+ * * Enforce a configurable delay between the two steps, with the ability to cancel in between.
+ *   - Even after the timer has passed to avoid locking it forever.
+ * * It is not possible to use another role to manage the `DEFAULT_ADMIN_ROLE`.
+ *
+ * Example usage:
+ *
+ * ```solidity
+ * contract MyToken is AccessControlDefaultAdminRules {
+ *   constructor() AccessControlDefaultAdminRules(
+ *     3 days,
+ *     _msgSender() // Explicit initial `DEFAULT_ADMIN_ROLE` holder
+ *    ) {}
+ *}
+ * ```
+ *
+ * NOTE: The `delay` is only configurable in the constructor to avoid issues related with handling
+ * delay management during the transfer is pending to be completed.
```

Comment:
No need to mention the reasons IMO.

```suggestion
 * NOTE: The `delay` can only be set in the constructor and is fixed thereafter.
```

Reply:
Fixed in https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4009/commits/fd64f5d4bb5038a3e037afb3a4a22c7c7bbf1dba since I couldn't accept this suggestion directly.

Solving since it'd have had the same effect